src/Spatial/Euclidean/PolyLine2D.cs

```diff
@@ -27,18 +26,18 @@ public class PolyLine2D : IEnumerable<Point2D>
         /// </summary>
         /// <param name="key"></param>
         /// <returns></returns>
-        public Point2D this[int key] => this._points[key];
+        public Point2D this[int key] => this.points[key];
 
         // Internal storage for the points
-        private List<Point2D> _points;
+        private readonly List<Point2D> points;
```

Comment:
this is a behavior change - it wasn't readonly before and the collection was exposed publically via the indexer

Reply:
The indexer is get only, how do you mean it is breaking?